src/utils.ts

```diff
@@ -233,6 +233,20 @@ export interface PaginationIteratorOptions {
     exclusiveStartId?: string;
 }
 
+export interface PaginationOptions {
```

Comment:
I'm thinking if now all the `XYZOptions` interfaces could extend this interface to reduce the duplicate code

see e.g. 

https://github.com/apify/apify-client-js/blob/7dfc5b354fe623798bf350f887fd565c8a524a99/src/resource_clients/webhook_dispatch_collection.ts#L36-L40